modules/openthread/platform/openthread-core-zephyr-config.h

```diff
@@ -446,4 +446,15 @@
 #define OPENTHREAD_CONFIG_PLATFORM_POWER_CALIBRATION_ENABLE 0
 #endif
 
+
+ /**
+ * @def OPENTHREAD_CONFIG_RADIO_STATS_ENABLE
+ *
+ * Enable support for Radio Statistics.
+ *
+ */
+#ifndef CONFIG_OPENTHREAD_RADIO_STATS_ENABLE
+#define OPENTHREAD_CONFIG_RADIO_STATS_ENABLE CONFIG_OPENTHREAD_CONFIG_RADIO_STATS_ENABLE
+#endif
```

Comment:
Inverse logic and copy-paste issue?

```suggestion
#ifdef CONFIG_OPENTHREAD_RADIO_STATS_ENABLE
#define OPENTHREAD_CONFIG_RADIO_STATS_ENABLE CONFIG_OPENTHREAD_RADIO_STATS_ENABLE
#endif
```